zeppelin-web/src/app/notebook/notebook.controller.js

```diff
@@ -956,8 +956,12 @@ angular.module('zeppelinWebApp').controller('NotebookCtrl',
     $scope.suggestions = [];
   };
 
-angular.element(document).click(function(){
-     angular.element('.userlist').hide();
-});
+  angular.element(document).click(function(){
+       angular.element('.userlist').hide();
+  });
+
+  angular.element(document).click(function(){
+    angular.element('.ace_autocomplete').hide();
```

Comment:
IMO we can have all such `hide()`s under one `angular.element(document).click(function()` for clearing such context menus, instead of defining a new one.